### ISSUE TITLE
Unknown CloudFormation resource type: AWS::Chatbot::SlackChannelConfiguration

### ISSUE THREAD

**Issue by RobertoTorino** [aws-cloudformation/cfn-lint#2025]
*cfn-lint version: (0.49.2)*

*Description of issue.*

AWS::Chatbot::SlackChannelConfiguration' is not recognized as a valid resource type.

* CloudFormation sample that generated the issue:
```yaml
SlackChannelConfig:
    Type: AWS::Chatbot::SlackChannelConfiguration
    Properties:
      ConfigurationName: !Sub "{${Environment}-securityhub-notifications"
      IamRoleArn: !GetAtt ChatBotManageIAMRole.Arn
      LoggingLevel: "ERROR"
      SlackChannelId: !Split [ ",", !Sub "${SlackChannelId}" ]
      SlackWorkspaceId: !Ref SlackWorkSpaceId
      SnsTopicArns:
        - !Ref SecurityHubSNSTopic
```
*Official documentation*
[AWS::Chatbot::SlackChannelConfiguration](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-chatbot-slackchannelconfiguration.html)

**Comment by PatMyron** [comment]
Are you linting for a particular region? Not able to reproduce this with the `us-east-1` default

**Comment by kddejong** [comment]
The only regions I don't see this resource in is `cn-north-1.json`, `cn-northwest-1.json`, `us-gov-east-1.json`, `us-gov-west-1.json`  based on the version you specified you should be okay in any other region.  That resource should have existed as of March 5, 2020

**Comment by kddejong** [comment]
I'm unable to replicate this issue using `cfn-lint --region eu-west-1 -- 2025.yaml` and the most recent version of cfn-lint.

**Comment by RobertoTorino** [comment]
tried both commands, get this:
`Traceback (most recent call last):
  File "c:\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Python39\Scripts\cfn-lint.exe\__main__.py", line 7, in <module>
  File "c:\python39\lib\site-packages\cfnlint\__main__.py", line 50, in main
    except cfnlint.core.CfnLintExitException as e:
AttributeError: module 'cfnlint.core' has no attribute 'CfnLintExitException'`

Latest cfn-lint with python 3.9.5

**Comment by kddejong** [comment]
https://github.com/aws-cloudformation/cfn-lint/blob/4a7af2bd53a9ad1ccaba3a509437c53102ade522/src/cfnlint/core.py#L25  We shouldn't be seeing that error as that exists.  

I wonder if an uninstall and re-install may be a valid next step.  I would try to make sure the folder is empty where cfn-lint gets uninstalled to make sure everything is removed.  `pip show cfn-lint` should show you the folder being doing the uninstall.

**Comment by kddejong** [comment]
If you are still having issues let us know and we can re-open the issue.
